test/op_shape_test.cpp

```diff
@@ -2572,29 +2572,29 @@ TEST_CASE(reshape_dyn_shape)
             }
         }
         migraphx::shape output{migraphx::shape::float_type, out_dyn_dims};
-        expect_shape(output, migraphx::make_op("reshape", {{"dims", new_shape}}), input);
+        expect_shape(output, migraphx::make_op("reshape_lazy", {{"dims", new_shape}}), input);
     }
 }
 
-TEST_CASE(reshape_multiple_non_fixed_error)
+TEST_CASE(reshape_lazy_multiple_non_fixed_error)
 {
     migraphx::shape input{migraphx::shape::float_type, {{1, 4}, {24, 24}, {10, 20}, {1, 1}}};
     std::vector<int64_t> new_shape = {0, 1, 0, 24};
-    throws_shape(migraphx::make_op("reshape", {{"dims", new_shape}}), input);
+    throws_shape(migraphx::make_op("reshape_lazy", {{"dims", new_shape}}), input);
 }
 
-TEST_CASE(reshape_fixed_ele_not_matching_error)
+TEST_CASE(reshape_lazy_fixed_ele_not_matching_error)
 {
     migraphx::shape input{migraphx::shape::float_type, {{1, 4}, {24, 24}, {10, 10}, {1, 1}}};
     std::vector<int64_t> new_shape = {0, 1, 5, 24};
-    throws_shape(migraphx::make_op("reshape", {{"dims", new_shape}}), input);
+    throws_shape(migraphx::make_op("reshape_lazy", {{"dims", new_shape}}), input);
 }
 
-TEST_CASE(reshape_non_fixed_not_matching_error)
+TEST_CASE(reshape_lazy_non_fixed_not_matching_error)
```

Comment:
we should cover some lines for the reshape copy op as well for the test coverage.

Reply:
Think we still need to make these tests

Reply:
I can add these in this afternoon

Reply:
Added

Reply:
test cases for the reshape copy need to be modified or deleted if required.